app/assets/stylesheets/tasks.scss

```diff
@@ -0,0 +1,68 @@
+html {
+	font-family: sans-serif;
+}
+
+body {
+	text-align: center;
+}
+
+th {
+	background-color: #008080;
+	padding: 5px;
+	color: #ffffff;
+}
+
+h1 {
+	color: #008080;
+	background-color:  #e0ebeb;
+	height: 40px;
+	font-weight:normal;
+	padding-top: 10px;
+}
+
+h2 {
+	color: #008080;
+}
+
+.task_button {
+	text-decoration: none;
+	background-color: #008080;
+	color: #ffffff;
+	padding: 5px;
+	border-radius: 6px;
+	float: left;
+	margin-bottom: 15px;
+	margin-right: 5px;
+}
+
+.task_button:hover {
+	background-color: #004d4d;
+}
+
+table {
+    border-collapse: collapse;
+}
+
+table, th, td {
+   border: 1px solid #e6ffff;
+}
+
+table {
+	width: 100%;
+}
+
+td {
+	padding: 5px;
+}
+
+tr {
+	background-color:  #ffffff;
+}
+
+tr:nth-child(odd) {
+	background-color: #e0ebeb;
+}
+
+// tr:hover {
+// 	background-color:  #004d4d;
+// }
```

Comment:
Check the tab settings in your text editor. RoR generally uses 2 spaces. Also, removing unnecessary comments before a PR is also standard practice to improve readability.